client/driver/executor/executor.go

```diff
@@ -92,14 +97,21 @@ type UniversalExecutor struct {
 	processExited chan interface{}
 	lre           *logging.FileRotator
 	lro           *logging.FileRotator
+	syslogServer  *logging.SyslogServer
+	syslogChan    chan *logging.SyslogMessage
 
 	logger *log.Logger
-	lock   sync.Mutex
+	cgLock sync.Mutex
+	rLock  sync.Mutex
```

Comment:
can cgLock and rLock have more descriptive names. Don't know what they are locking. Usually group the lock around the variables it is locking